engine/server/engine/server/engine_connect_server_service.go

```diff
@@ -0,0 +1,133 @@
+package server
```

Comment:
This is the new way of creating the server; need to do the same api_container. pretty straight-forward imo. The complexity is with streams.